packages/edit-site/src/components/style-book/button-examples.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import { __experimentalGrid as Grid } from '@wordpress/components';
+import { View } from '@wordpress/primitives';
+import {
+	BlockList,
+	privateApis as blockEditorPrivateApis,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { ExperimentalBlockEditorProvider, useGlobalStyle } = unlock(
+	blockEditorPrivateApis
+);
+
+const BUTTON_STATES = [
+	{
+		key: 'default',
+		title: __( 'Button' ),
+	},
+	{
+		key: ':active',
+		title: __( 'Button (Active)' ),
+	},
+	{
+		key: ':any-link',
+		title: __( 'Button (Any Link)' ),
+	},
+	{
+		key: ':focus',
+		title: __( 'Button (Focus)' ),
+	},
+	{
+		key: ':hover',
+		title: __( 'Button (Hover)' ),
+	},
+	{
+		key: ':link',
+		title: __( 'Button (Link)' ),
+	},
+	{
+		key: ':visited',
+		title: __( 'Button (Visited)' ),
+	},
+];
+
+function ButtonExamples() {
+	const [ elementsButton ] = useGlobalStyle( 'elements.button' );
+	const blocks = BUTTON_STATES.map( ( { key } ) => {
+		const styles =
+			( key !== 'default' ? elementsButton[ key ] : elementsButton ) ||
+			{};
+		return createBlock( 'core/button', {
+			text: __( 'Call to Action' ),
```

Comment:
Better than the above still, instead of call to action, make the button text the state. The downside is that the buttons' widths will vary, but the CSS could be tweaked a bit e.g., 

<img width="812" alt="Screenshot 2024-12-06 at 1 48 09 pm" src="https://github.com/user-attachments/assets/2e0d55d1-4ba1-4058-9f77-840b1f64a8b2">

Anyway, just an idea.